src/api/app/models/workflow/step/rebuild_package.rb

```diff
@@ -6,7 +6,7 @@ class Workflow::Step::RebuildPackage < Workflow::Step
   attr_reader :project_name, :package_name
 
   def call
-    return if workflow_run.closed_merged_pull_request? || workflow_run.reopened_pull_request?
+    return if workflow_run.closed_merged_pull_request? || workflow_run.reopened_pull_request? || workflow_run.unlabeled_pull_request?
```

Comment:
Why skip this step for labeling?

Reply:
This is only skipped for `unlabeled` action. There is nothing more to do in this step for `unlabeled` action. When the event is `labeled` we do execute this whole method.